pkg/extensions/search/common/oci_layout.go

```diff
@@ -414,7 +414,12 @@ func (olu BaseOciLayoutUtils) GetExpandedRepoInfo(name string) (RepoInfo, error)
 			continue
 		}
 
-		vendor := olu.GetImageVendor(imageConfigInfo)
+		// vendor := olu.GetImageVendor(imageConfigInfo)
+		vendor := GetVendor(manifest.Annotations)
+		if vendor == "" {
+			vendor = GetVendor(imageConfigInfo.Config.Labels)
+		}
+
```

Comment:
We probably want NewestImage set on RepoSummary as well. And that would include the new label data as well. Actually, GetExpandedRepoInfo would be used when you open the repo product page, which is one place where we need the new data.